recipes/docopt_subcommands/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "docopt_subcommands" %}
+{% set version = "2.2.0" %}
+{% set hash_type = "sha256" %}
+{% set hash = "c5f2f47f4dbbf4e633752765146a9880e566b2e41906b3c728461389eb04a48c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
+  skip: True
```

Comment:
This probably should have a selector, right? Also if there is a platform to skip, what is the reason?